### PR TITLE
Show sidebar

### PR DESCRIPTION
**Pull request by yuzi03301114**

### CODE REVIEW

File: src/components/LayoutHome.vue [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+        <img src="../assets/images/svgs/align-right.svg" alt="Image" class="w-[24px] h-[24px] mb-4" />
+      </botton>
       </div>
+     <div class="flex items-center mt-10">
+       <!-- <h2 class="text-[30px] leading-[48px]">Memo</h2> -->
      </div>
-      <div class="ml-80 w-3/4" :class="showComponent ? 'translate-x-[-140px]' : 'translate-x-0'">
-       <slot></slot>
-     </div>
+      <ul class=" fixed h-screen shadow-lg w-[max-content] z-[1000] transition-all translate-x-[-360px] md:translate-x-0'" :class="showComponent ? 'translate-x-[-360px]' : 'translate-x-0' ">
```
Comment (zemarr) [inline src/components/LayoutHome.vue]: This is not what I asked you to do with the sidebar. The sidebar should not change from the original design and can only be toggled when the device is a tablet or mobile.
Revert the sidebar to the original sidebar, this one is unacceptable.

File: src/components/LayoutHome.vue [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-          <img src="../assets/images/svgs/align-right.svg" alt="Image" class="w-[24px] h-[24px] mb-4" />
-        </botton>
-        </div>
-       <div class="flex items-center mt-10">
-         <!-- <h2 class="text-[30px] leading-[48px]">Memo</h2> -->
-       </div>
-        <ul class="w-[max-content] z-[1000] transition-all translate-x-[-360px] md:translate-x-0'" :class="showComponent ? 'translate-x-[-360px]' : 'translate-x-0'">
-        <li v-for="item in keyList" :key="item.id" @click="goAnchor(item)" class="cursor-pointer text-slate-500 text-[18px] font-normal leading-[24px] rounded-[4px] mb-[8px] hover:underline hover:text-black">{{ item.name }}</li>
-       </ul>
+      <botton class="fixed right-430 cursor-pointer" @click="showComponent = !showComponent" >
```
Comment (zemarr) [inline src/components/LayoutHome.vue]: You can't have this button in a fixed position. Make it absolute position instead

**Comment by yuzi03301114** [comment]
OK I got it. I will fix it. Sorry about that.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by zemarr** [comment]
LGTM

**Pull request merged**